apps/builder/app/builder/features/style-panel/shared/use-style-data.ts

```diff
@@ -18,6 +18,7 @@ import {
 import { useStyleInfo } from "./style-info";
 import { serverSyncStore } from "~/shared/sync";
 import { $ephemeralStyles } from "~/canvas/stores";
+import { $isEphemeralUpdateInProgress } from "~/builder/shared/nano-states";
```

Comment:
would checking $ephemeralStyles work for you?

Reply:
actually it might